yarn.lock

```diff
@@ -3519,6 +3519,11 @@ is-wsl@^1.1.0:
   resolved "https://registry.yarnpkg.com/is-wsl/-/is-wsl-1.1.0.tgz#1f16e4aa22b04d1336b66188a66af3c600c3a66d"
```

Comment:
I included the changes to `lock.json` that should have been added with PR #871. Can remove this if necessary. Maybe this fix could be backported if this is removed?

Reply:
I think you're right so I would remove the changes in this file.

Reply:
Reverted the changes to the lock file